pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -742,7 +742,7 @@ def _add_disk_overview(self, disk, box):
         # We don't want to display the whole huge WWID for a multipath device.
         # That makes the DO way too wide.
         if isinstance(disk, MultipathDevice):
-            desc = disk.wwid.split(":")
+            desc = disk.wwn.split(":")
             description = ":".join(desc[0:3]) + "..." + ":".join(desc[-4:])
```

Comment:
Do wwn and wwid have the same value or the format of the value? Otherwise, the parsing might not work.

Reply:
Good catch. The formatting is not identical.